Resources/Prototypes/_NF/Recipes/Lathes/electronics.yml

```diff
@@ -29,3 +29,11 @@
   materials:
     Steel: 400
     Glass: 900
+
+- type: latheRecipe
+  id: TilePrinterNFMachineCircuitboard
+  result: TilePrinterNFMachineCircuitboard
+  completetime: 4
+  materials:
+    Steel: 150
+    Glass: 150
```

Comment:
increase the material cost of these or you'll get people printing them for money, check some of the other values in here for example.